homeassistant/components/ezviz/camera.py

```diff
@@ -303,3 +307,13 @@ def perform_set_alarm_detection_sensibility(
             )
         except (HTTPError, PyEzvizError) as err:
             raise PyEzvizError("Cannot set detection sensitivity level") from err
+
+        ir.async_create_issue(
+            self.hass,
+            DOMAIN,
+            "manual_migration",
```

Comment:
I don't think this is a great key name to use for this, maybe `detection_sensibility_deprecated` or something?

Reply:
Could potentially drive a github action in the future from this. Would service_depreciation_detection_sensibility be too long? Would then have a standard prefix with service name.

Reply:
It's up to you, let's just not use "manual_migration" because it's a really poor match with the actual issue 👍 
The reason we need to be careful with these keys is that they're used for translations, meaning changing or reusing a key is a big hassle.

Reply:
Done.